docs/user_guide/transformation/ArcsinTransformer.rst

```diff
@@ -0,0 +1,79 @@
+.. _reciprocal:
+
+.. currentmodule:: feature_engine.transformation
+
+ArcsinTransformer
+=====================
+
+The :class:`ArcsinTransformer()` applies the arcsin transformation to
+numerical variables.
+
+The :class:`ArcsinTransformer()` only works with numerical variables with values between -1 and +1. If the variable contains a value outside of this range, the transformer will raise an error.
+
+Let's load the house prices dataset and  separate it into train and test sets (more
+details about the dataset :ref:`here <datasets>`).
+
+.. code:: python
+
+	import numpy as np
+	import pandas as pd
+	import matplotlib.pyplot as plt
+	from sklearn.model_selection import train_test_split
+
+	from feature_engine import transformation as vt
+
+	# Load dataset
+	data = data = pd.read_csv('houseprice.csv')
+
+	# Separate into train and test sets
+	X_train, X_test, y_train, y_test =  train_test_split(
+		    data.drop(['Id', 'SalePrice'], axis=1),
+		    data['SalePrice'], test_size=0.3, random_state=0)
+
+Now we want to apply the arcsin transformation to 2 variables in the dataframe:
+
+.. code:: python
+
+	# set up the variable transformer
+	tf = vt.ArcsinTransformer(variables = ['LotArea', 'GrLivArea'])
```

Comment:
are these variables between -1 and 1? I don't remember.

For sure, the California housing dataset from sklearn has a decimal variable we can use for the demo, which also gets a nice gaussian distribution after the arcsin, so it is kind of the perfect example 📦 

I normally run the code I write in these examples in a jupyter notebook, and then copy and paste the output here.

Reply:
```python
# Load dataset

cal_housing = fetch_california_housing()
X = pd.DataFrame(cal_housing.data, columns=cal_housing.feature_names)
# divide by 50 to obtain values between -1 and +1
X['Latitude'] = X['Latitude'] / 50
y = cal_housing.target
```
I loaded the California housing dataset but I did not find a decimal variable, so I just take one of the column and divide all his value by 50 to make sure to have values between -1 and +1

Reply:
hmm, that is not ideal, we'd like to make demos as "real" as possible. Would you be able to check if the [breast cancer dataset](https://scikit-learn.org/stable/datasets/toy_dataset.html#breast-cancer-wisconsin-diagnostic-dataset) has some suitable variables.

Alternatively, the [kdd99 dataset](https://scikit-learn.org/stable/datasets/real_world.html#kddcup-99-dataset), which [variable description is here](https://kdd.ics.uci.edu/databases/kddcup99/task.html) has some % variables (see bottom table), which if not in decimal scale already, we could divide by 100 and that would make more sense.

Worse case scenario, the [iris dataset's](https://scikit-learn.org/stable/datasets/toy_dataset.html#iris-plants-dataset) variables are in cm, if we divide by 10, they would be decimal and it is still a valid measure.

The idea of the demo is to show that the variable originally was skewed, and after the transformation it is less skewed. So we would need to play around with the datasets, to see which one fits best. Thank you so much!!!